File: sdk/api/src/chan/content/ThreadRedirectException.java
package chan.content;

import chan.library.api.BuildConfig;

public final class ThreadRedirectException extends Exception {
	public ThreadRedirectException(String boardName, String threadNumber, String postNumber) {
		BuildConfig.Private.expr(boardName, threadNumber, postNumber);
	}

	public ThreadRedirectException(String threadNumber, String postNumber) {
		BuildConfig.Private.expr(threadNumber, postNumber);
	}
}

File: sdk/api/src/chan/content/ThreadRedirectException.kt
package chan.content

import chan.library.api.BuildConfig

class ThreadRedirectException : Exception {
    constructor(boardName: String, threadNumber: String, postNumber: String) {
        BuildConfig.Private.expr<Any>(boardName, threadNumber, postNumber)
    }

    constructor(threadNumber: String, postNumber: String) {
        BuildConfig.Private.expr<Any>(threadNumber, postNumber)
    }
}
